source/operation/vm_management/vm_instances.rst

```diff
@@ -599,6 +604,87 @@ These actions can be deleted or edited using the ``onevm update`` command. The t
       ID="1",
       TIME="1379938500" ]
 
+Relative actions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Like in puntual actions, to schedule this kind of actions, also use the option ``--schedule``, but this command also needs more options, these options define the relativity of the action. The options are:
```

Comment:
To schedule relative actions also use the option ``--schedule``. however this command also needs more options to define the relativity of the action.